npm-global-without-sudo.md

```diff
@@ -20,12 +20,15 @@ prefix=${HOME}/.npm-packages
 
 ```sh
 NPM_PACKAGES="${HOME}/.npm-packages"
+#add NPM_PACKAGES to NODE_PATH
+export NODE_PATH="NODE_PATH:$NPM_PACKAGES/lib/node_modules"
```

Comment:
Oddly enough, it does work without '$." I had it exactly this way in my .bashrc and it worked okay. It does make more sense to have it there, though, and I tested the change to make sure it works. So I'm making the change as suggested.